### ISSUE TITLE
Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by renilbabu03** [angular-ui/angular-google-maps#2004]
I am using ng-map in my application. It is working fine in many places but in some places it is throwing the error "Uncaught RangeError: Maximum call stack size exceeded". I have tried different solution in stackoverflow but of no effect.

**Comment by ankitjs** [comment]
same error.....

**Comment by saurabhSiddhu** [comment]
Have you created google api id? Can you explain bit more which api you are using?

**Comment by renilbabu03** [comment]
Yes I have created one . Its working in other panels but not on this particular one. What may be the reason.?

**Comment by renilbabu03** [comment]
![maximumcallstack](https://cloud.githubusercontent.com/assets/18526147/25176464/cd9f3a34-251b-11e7-9168-1ccc64ae8574.png)

**Comment by renilbabu03** [comment]
I am using ng-map in modal. All places except the modal , the map is working, but not in the modal. How to fix it.?
